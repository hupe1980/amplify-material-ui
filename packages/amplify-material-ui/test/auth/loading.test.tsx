import * as React from 'react';
import { render } from '@testing-library/react';
import { Loading } from '../../src';
import { withTheme } from '../helpers/withTheme';

describe('loading', () => {
  it('should be rendered correctly', () => {
    const { asFragment } = render(withTheme(<Loading />));
    expect(asFragment()).toMatchSnapshot();
  });
});
