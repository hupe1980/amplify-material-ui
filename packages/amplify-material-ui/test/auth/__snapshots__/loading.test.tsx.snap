// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading should be rendered correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthXs css-nzinn5-MuiContainer-root"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 makeStyles-paper-2 css-1ps6pg7-MuiPaper-root"
    >
      <span
        class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorSecondary makeStyles-progress-1 css-1szo49z-MuiCircularProgress-root"
        role="progressbar"
        style="width: 40px; height: 40px;"
      >
        <svg
          class="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
          viewBox="22 22 44 44"
        >
          <circle
            class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
            cx="44"
            cy="44"
            fill="none"
            r="20.2"
            stroke-width="3.6"
          />
        </svg>
      </span>
    </div>
  </div>
</DocumentFragment>
`;
